### ISSUE TITLE
Suggest to use Map instead of plain object for faster reads

### ISSUE THREAD

**Issue by brainysmurf** [joshsny/SpeedStore#3]
```js
const memcache = new Map(); 
// instead of 
const memcache = {};
```

You'll have to refactor a bit, but map reads are way faster than associate object reads.

I don't know how to do that in typescript though.

**Comment by joshsny** [comment]
Great idea, thanks for that suggestion. I'll refactor it soon!

**Comment by joshsny** [comment]
This was implemented in #8, thanks for the suggestion!
